### PR TITLE
Gentoo

### PR DESCRIPTION
**Pull request by drwilly**
various gentoo-related completion changes

### CODE REVIEW

File: share/completions/ebuild.fish [hunk 1]
```diff
@@ -12,6 +12,7 @@ complete -c ebuild -l skip-manifest	-d "Skip all manifest checks"
 
 ## Subcommands
 complete -c ebuild -n '__fish_seen_ebuild_arg' -xa 'help'        -d "Show help"
+complete -c ebuild -n '__fish_seen_ebuild_arg' -xa 'pretend'     -d "Run pkg_pretend()"
```
Comment (faho) [inline share/completions/ebuild.fish]: What's a `pkg_pretend()`? Is it edible?

File: share/completions/emaint.fish [hunk 2]
```diff
@@ -18,11 +18,11 @@ complete -c emaint -n '__fish_use_subcommand' -xa 'sync'        -d 'Check repos.
 complete -c emaint -n '__fish_use_subcommand' -xa 'world'       -d 'Check and fix problems in the world file'
 
 ## Local opts
-# cleanlogs
```
Comment (faho) [inline share/completions/emaint.fish]: Is `cleanlogs` no longer a thing?

File: share/completions/emaint.fish [hunk 3]
```diff
@@ -18,11 +18,11 @@ complete -c emaint -n '__fish_use_subcommand' -xa 'sync'        -d 'Check repos.
...
-complete -c emaint -n '__fish_seen_subcommand_from cleanlogs' -s t -l time      -d "Delete logs older than NUM of days" \
+# logs
+complete -c emaint -n '__fish_seen_subcommand_from logs' -s t -l time      -d "Delete logs older than NUM of days" \
```
Comment (faho) [inline share/completions/emaint.fish]: That `of` feels weird to me - `Delete logs older than NUM days`.

File: share/completions/ebuild.fish (hunk 1 above)
Comment (faho) [reply share/completions/ebuild.fish]: I'm assuming this is a "--dry-run" option, that just pretends to do the operation?

Comment (drwilly) [reply share/completions/ebuild.fish]: Yes, kinda: https://devmanual.gentoo.org/ebuild-writing/functions/pkg_pretend/index.html

File: share/completions/emaint.fish (hunk 2 above)
Comment (drwilly) [reply share/completions/emaint.fish]: nope

File: share/completions/emaint.fish (hunk 3 above)
Comment (drwilly) [reply share/completions/emaint.fish]: `of`?

Comment (faho) [reply share/completions/emaint.fish]: In the description: "Delete logs older than NUM **of** days"

Comment (drwilly) [reply share/completions/emaint.fish]: I swear I read that line like eight times and did not see an `of` in there. You are right though, the wording is weird.

**Comment by faho** [comment]
Merged, thanks!

**Pull request merged**